### PR TITLE
docs: update docstrings of JinaDocumentEmbedder and JinaTextEmbedder

### PR DESCRIPTION
**Pull request by DresAaron**
Updated function doc

### CODE REVIEW

File: integrations/jina/src/haystack_integrations/components/embedders/jina/document_embedder.py [hunk 1]
```diff
@@ -62,6 +62,14 @@ def __init__(
                              to keep the logs clean.
         :param meta_fields_to_embed: List of meta fields that should be embedded along with the Document text.
         :param embedding_separator: Separator used to concatenate the meta fields to the Document text.
+        :param task: The downstream task for which the embeddings will be used.
+            The model will return the optimized embeddings for that task.
+            Check the list of available models on [Jina documentation](https://jina.ai/embeddings/).
```
Comment (julian-risch) [inline integrations/jina/src/haystack_integrations/components/embedders/jina/document_embedder.py:67]:
```suggestion
            Check the list of available tasks on [Jina documentation](https://jina.ai/embeddings/).
```

File: integrations/jina/src/haystack_integrations/components/embedders/jina/text_embedder.py [hunk 2]
```diff
@@ -52,6 +52,14 @@ def __init__(
             Check the list of available models on [Jina documentation](https://jina.ai/embeddings/).
         :param prefix: A string to add to the beginning of each text.
         :param suffix: A string to add to the end of each text.
+        :param task: The downstream task for which the embeddings will be used.
+            The model will return the optimized embeddings for that task.
+            Check the list of available models on [Jina documentation](https://jina.ai/embeddings/).
```
Comment (julian-risch) [inline integrations/jina/src/haystack_integrations/components/embedders/jina/text_embedder.py:57]:
```suggestion
            Check the list of available tasks on [Jina documentation](https://jina.ai/embeddings/).
```

**Review by julian-risch — CHANGES_REQUESTED** [review]
Hello @DresAaron thanks for updating the docstrings! One change request/question: the the newly introduced parameters are only supported for jina-embeddings-v3 correct? As users might still use `jina-embeddings-v2-base-en`, we should mention in the docstrings that support for these parameters is limited to `jina-embeddings-v3`.
In addition I suggested changing `models` to `tasks` in the docstring about `task`. Otherwise the PR looks good to me. 👍

**Review by julian-risch — APPROVED** [review]
Looks good to me! 👍 Thanks for addressing the change requests so quickly.

**Comment by DresAaron** [comment]
> Looks good to me! 👍 Thanks for addressing the change requests so quickly.

@julian-risch Thank you so much for you help! Could you merge it or we have to wait?

**Pull request merged**